src/Adapters/DiagnosticSource/DiagnosticSourceListener.csproj

```diff
@@ -0,0 +1,76 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Company>Microsoft</Company>
+    <Copyright>Copyright © Microsoft. All Rights Reserved.</Copyright>
```

Comment:
Minor: I remember guidelines saying that copyright should be "© Microsoft Corporation. All rights reserved."

Reply:
I'll address it in separate review. Will check the guidelines